plutarch.cabal

```diff
@@ -1,39 +1,32 @@
 cabal-version:      2.4
 name:               plutarch
 version:            1.0.0
-
-author: Las Safin <me@las.rs>
-license: MIT
-
+author:             Las Safin <me@las.rs>
+license:            MIT
 extra-source-files: README.md
 
 flag development
   description: Enable tracing functions within plutarch.
-  manual: True
-  default: False
+  manual:      True
```

Comment:
Why did you change all the formatting here?

Reply:
cabal-fmt did it